### DISCUSSION TITLE
Whats planned for the future?

### DISCUSSION THREAD

**Discussion by AverageVirusTotalEnjoyer** [winstxnhdw/lc-hax#121] [Q&A]

**Comment by AverageVirusTotalEnjoyer** [comment]
@winstxnhdw

**Comment by winstxnhdw** [comment]
ESP is next.

**Comment by yujispin990** [comment]
> ESP is next.

Could i suggest adding a Spectators' List which lists all players in the lobby as a text on a small display probably somewhere in the top corner of the screen and when a player dies the color of their name changes to indicate that they have died. As well as, adding "Voice Chat Indicator" like a Volume Icon or a text saying E.g "Talking" to the side of their name to indicate which one is talking.

**Comment by AverageVirusTotalEnjoyer** [comment]
@yujispin990 there are mods on thunderstore that allows you to do that

**Reply by yujispin990** [reply]
Requires every player in the lobby to have the mod installed for you to use it.
